README.md

```diff
@@ -39,7 +39,9 @@ under the repository. You can launch the tests manually for
 each environment.
 
 A nightly test on the production environment only is also run
-at midnight every day automatically.
+at midnight every day automatically. This nightly build uses the latest
+tag on the master branch. If your changes do not depend on
+any repo's release you should create a new tag after merging.
```

Comment:
`any other`? In fact, maybe rephrase to `If your changes do not depend on any changes in another repo,`

Reply:
changed